compiler/damlc/daml-ide-core/dlint.yaml

```diff
@@ -193,7 +193,7 @@
     - warn: {lhs: intersectBy (==), rhs: intersect}
     - warn: {lhs: maximumBy compare, rhs: maximum}
     - warn: {lhs: minimumBy compare, rhs: minimum}
-    - warn: {lhs: nubBy (==), rhs: nub}
+    - warn: {lhs: dedupBy compare, rhs: dedup}
```

Comment:
Is it intentional that the `==` has become `compare`?

Reply:
Ok, answering myself, I expect that it if you are not supplying a custom predicate then it is.

Reply:
Our `dedupBy` avoids being quadratic by relying on `Ord`. That’s why it needs `compare` instead of `==`.